packages/hackathon/src/scenes/ProjectsScene/components/ProjectView.tsx

```diff
@@ -32,25 +32,35 @@ import type { IProjectProps } from '../../../models'
 import { getTechnologies } from '../../../data/hack_session/selectors'
 import { getMembers, techDescriptions } from '../../utils'
 
-interface ProjectViewProps {
-  project: IProjectProps
-}
+type ProjectViewProps = Pick<
+  IProjectProps,
+  | 'description'
+  | 'technologies'
+  | 'members'
+  | 'title'
+  | 'project_type'
+  | 'contestant'
+>
 
-export const ProjectView: FC<ProjectViewProps> = ({ project }) => {
+export const ProjectView: FC<ProjectViewProps> = ({
+  description,
+  technologies,
+  members,
+  title,
+  project_type,
+  contestant,
+}) => {
```

Comment:
What prompted using individual properties rather than the project object? This makes any future changes we make to project more fragile, requiring updates in multiple places. What is the benefit of breaking the encapsulation?

Reply:
I see where you made this change for judging. See my comment there.